internal/website/content/tutorials/deploy-gcp.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Tutorial: Deploy to GCP"
+linkTitle: "Deploy to GCP"
+date: 2019-03-20T09:47:11-07:00
+weight: 3
+---
+
+In this tutorial, we will deploy an existing Go CDK application called
+Guestbook to Google Cloud Platform (GCP).
+
+<!--more-->
+
+{{< snippet "tutorials/deploy/guestbook.md" >}}
+
+## Prerequisites
+
+You will need to install the following software for this tutorial:
+
+-   [Git](https://git-scm.com/)
+-   [Go](https://golang.org/doc/install)
+-   [Wire](https://github.com/google/wire/blob/master/README.md#installing)
+-   [Docker Desktop](https://docs.docker.com/install/)
```

Comment:
Do we need docker for the cloud deployment? Maybe mention that this is optional?

Reply:
Nice, I forgot that the GCP version uses Cloud Build. Hooray for splitting the docs.

Reply:
Wait, I'm a little confused now. Is it true that we only need Docker to run MySql locally? Or do we also need it to talk to the AWS MySql thingy?

If the former, it's worth saying so in the parens for the list of prerequisites - since it's not clear why Docker is needed (the list of prerequisites is already formidable)

Reply:
🤦‍♂️ Oh right.

For GCP, you only need it for running locally; you don't need it for deploy. All the other guides need it for both.